src/MongoDB.Driver.Core/ServerErrorCode.cs

```diff
@@ -38,6 +38,7 @@ internal enum ServerErrorCode
         SocketException = 9001,
         UnknownReplWriteConcern = 79,
         UnsatisfiableWriteConcern = 100,
-        WriteConcernFailed = 64
+        WriteConcernFailed = 64,
+        MaxTimeMSExpired= 50
```

Comment:
Add space before `=`.

Reply:
It's fine to add this value to the enum, but I don't see it actually used anywhere.